app/src/main/java/chat/rocket/android/helper/ServerPolicyHelper.java

```diff
@@ -71,6 +71,14 @@ private static String removeProtocol(String hostname) {
     return hostname.replace("http://", "").replace("https://", "");
   }
 
+  private static String removeTrailingSlash(String hostname) {
+    if (hostname.charAt(hostname.length() - 1) != '/') {
```

Comment:
`hostname.replaceAll("/+$", "")` removes the slashes at the tail of hostname.
Is this condition really needed?